src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/MarkdownExtension.kt

```diff
@@ -0,0 +1,81 @@
+package nl.avisi.structurizr.site.generatr.site.views
+
+import kotlinx.html.*
+import nl.avisi.structurizr.site.generatr.site.asUrlToFile
+import nl.avisi.structurizr.site.generatr.site.model.PageViewModel
+
+fun HEAD.markdownAdmonitionStylesheet(viewModel: PageViewModel) {
+    link(
+        rel = "stylesheet",
+        href = "../" + "/admonition.css".asUrlToFile(viewModel.url)
+    )
+}
+
+fun BODY.markdownAdmonitionScript(viewModel: PageViewModel) {
+    script(
+        type = ScriptType.textJavaScript,
+        src = "../" + "/admonition.js".asUrlToFile(viewModel.url)
+    ) { }
+}
+
+fun HEAD.katexStylesheet() {
+    // loading KaTeX as global on a webpage: https://katex.org/docs/browser.html#loading-as-global
+    // <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.6/dist/katex.min.css" integrity="sha384-mXD7x5S50Ko38scHSnD4egvoExgMPbrseZorkbE49evAfv9nNcbrXJ8LLNsDgh9d" crossorigin="anonymous">
```

Comment:
Please remove this line. It's just a commented-out copy of line 26.

Reply:
done